contracts/hyperlane-mailbox/src/main.sw

```diff
@@ -38,6 +33,12 @@ const ZERO_ID: ContractId = ContractId {
     value: 0x0000000000000000000000000000000000000000000000000000000000000000,
 };
 
+configurable {
```

Comment:
this is like `immutable`?

Reply:
yup exactly, you can set it at deploy time (example https://github.com/hyperlane-xyz/fuel-contracts/pull/52/files/23415c91c92850595b144b6a7b0c5b2575893361#diff-ec80a515b886e4b561ad16c6bd0f10852a4d8beda8e09f3be4bd076bb61743b5R64-R75) but also need to provide a default